www/xcaddy/src/opnsense/service/templates/OPNsense/XCaddy/xcaddy_build_config.json

```
@@ -0,0 +1,17 @@
+{
+    "version": "v2.10.0",
```

fine by me, just that having this and commits and module URLs hardcoded will mean constant maintenance in the future

---

Thats a good suggestion, I will think about this some. Probably best to remove this.

---

You could make this a json.in file and use:

```suggestion
    "version": "v%%PLUGIN_VERSION%%",
```

and we add PLUGIN_VERSION to the words to be replace at build time. Then you can steer this via the plugin's version, but in the end the GitHub commit stuff is still the same problem not easily changed in this way.

---

I have opted to always build latest, as building a specific version might also put constraint on the commit hashes of certain modules.

Now, everything is generalized and will not need much maintenance in the future.